src/core/store/modules/tokens/tokens.reducer.ts

```diff
@@ -82,8 +87,9 @@ const tokensReducer = createReducer(tokensInitialState, (builder) => {
     .addCase(getTokens.fulfilled, (state, { payload: { tokensData } }) => {
       const tokenAddresses: string[] = [];
       tokensData.forEach((token) => {
-        state.tokensMap[token.address] = token;
-        tokenAddresses.push(token.address);
+        // TODO: Should lowercase be handled here or in subgraph?
+        state.tokensMap[token.address.toLowerCase()] = token;
```

Comment:
From Kiba: 

instead of lowercasing all address in tokenMap, modify utils/lines.ts and use ethers.utils.getAddress(address) before saving as a checksum address (

Normal address --> full lowercase
checksum address --> same address, but has mixture of upper and lowercase (cryptograpically less likely to mess up inputs)

Ethers.js has a utility to convert normal addresses to checksum addresses